models/public/nfnet-f0/model.yml

```diff
@@ -0,0 +1,235 @@
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/classifier.py
+  - name: timm/models/layers/config.py
+    size: 3069
+    sha256: 3676b6ecffc1d5ccb8a1bb3831ae7fb1de559577bfb021188eff6db7234005c1
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/config.py
+  - name: timm/models/layers/create_act.py
+    size: 3904
+    sha256: a652617d6228e9f06cdbca6514f6deb7953a628e80a0a132955c89d2bb8aeace
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/create_act.py
+  - name: timm/models/layers/create_attn.py
+    size: 1490
+    sha256: f5f50a43ac8c52240c58ea2c5dcfb94e1fcfad77350a8fd4e286e12743140f6d
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/create_attn.py
+  - name: timm/models/layers/drop.py
+    size: 6938
+    sha256: 99cd8bf3d9e503ca4633264141fda078662e4bb7e94367a17c7a5fda646eab4d
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/drop.py
+  - name: timm/models/layers/helpers.py
+    size: 724
+    sha256: 6c60bfaed9d9d53f15bf6f5f9655bac581f0e61f8ac2ef41bd9f1d216cd9dcbf
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/helpers.py
+  - name: timm/models/layers/linear.py
+    size: 743
+    sha256: 6da4b65a9974bd810b9ef86743a2f0eb98d5a2d689cf98866eba0927d2662113
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/linear.py
+  - name: timm/models/layers/padding.py
+    size: 2167
+    sha256: 0636edc499ae8be0f27aba19a2160c751023e66478a3aa56816d388a68b78432
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/padding.py
+  - name: timm/models/layers/pool2d_same.py
+    size: 2969
+    sha256: 0b4b777bb51fb0e4ed4c526ce3f1b29eacdedd73e0262b42c5ad676a3036474d
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/pool2d_same.py
+  - name: timm/models/layers/se.py
+    size: 2294
+    sha256: a00fcd253ea07c960a0d71e8c12fb2d9e53cf282d1658d406b55ea8ccd2754e7
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/se.py
+  - name: timm/models/layers/std_conv.py
+    size: 6578
+    sha256: 91cc840771cdab22f814f9b1a90b750629cb5f334cbd9b3c0f3e10d526e2e440
+    source: https://raw.githubusercontent.com/rwightman/pytorch-image-models/740f32c96ad6da35e5b185d4981ec9ae867073cc/timm/models/layers/std_conv.py
```

Comment:
I see that this project is on PyPI. Maybe it would be easier to download a release wheel and unpack it?

Reply:
Done